include/dba.php

```diff
@@ -1083,15 +1084,40 @@ public static function update($table, $fields, $condition, $old_fields = array()
 		return self::e($sql, $params);
 	}
 
+	/**
+	 * Retrieve a single record from a table and returns it in an associative array
+	 * 
+	 * @brief Retrieve a single record from a table
+	 * @param string $table
+	 * @param array  $fields
+	 * @param array  $condition
+	 * @param array  $params
+	 * @return bool|array
+	 * @see dba::select
+	 */
+	public static function selectOne($table, array $fields = [], array $condition = [], $params = [])
```

Comment:
Why have you called the function "selectOne" and not "selectFirst" (which would fit with the existing function name "fetch_first")?